GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -421,6 +446,26 @@ bool ExporterHelper::ExportElectronFiles(const gd::Project &project,
             .FindAndReplace("\"GDJS_GAME_VERSION\"", jsonVersion)
             .FindAndReplace("\"GDJS_GAME_MANGLED_NAME\"", jsonMangledName);
 
+    gd::String packages = "";
+
+    for(std::shared_ptr<gd::PlatformExtension> extension : project.GetCurrentPlatform().GetAllPlatformExtensions()) { //TODO Add a way to select only used Extensions
+      for (gd::DependencyMetadata dependency: extension->GetAllDependencies()) {
+        if (dependency.GetDependencyType() == "npm") {
+          if(dependency.GetVersion() == "-1") {
+            gd::LogError("Latest Version (-1) not available for NPM dependencies, dependency " + dependency.GetName() + " is not exported.");
+            continue;
+          }
+          packages += "\n\t\"" + dependency.GetExportName() + "\": \"" + dependency.GetVersion() + "\",";
```

Comment:
Use only `\n` for new lines, not `\n\t` (unix style).

Reply:
the t is a tab for indentation not for the new line

Reply:
Ah silly me! Forget about this, looks good then!